environments/staging/terraform.tfvars

```diff
@@ -0,0 +1,6 @@
+#------------------------------------
```

Comment:
Is it worth having a separate tfvars file? Could just have a default. Up to you though. Will make more sense when we have different staging/prod environments.